docs/source/index.rst

```diff
@@ -25,7 +25,7 @@ Guide
    databases
    running-autometa
    examining-results
-   Scripts/scripts
+   scripts_2/script_main
```

Comment:
It is unclear to me as to why we should append `_2` instead of keeping the current `Scripts/scripts` organization. We could even go with `Scripts/Usage`?